docs/source/models.rst

```diff
@@ -784,6 +784,60 @@ method to load MLflow Models with the ``prophet`` model flavor in native prophet
 
 For more information, see :py:mod:`mlflow.prophet`.
 
+Pmdarima (``pmdarima``)
+^^^^^^^^^^^^^^^^^^^^^^^
+The ``pmdarima`` model flavor enables logging of `pmdarima models <http://alkaline-ml.com/pmdarima/>`_ in MLflow
+format via the :py:func:`mlflow.pmdarima.save_model()` and :py:func:`mlflow.pmdarima.log_model()` methods.
+These methods also add the ``python_function`` flavor to the MLflow Models that they produce, allowing the
+model to be interpreted as generic Python functions for inference via :py:func:`mlflow.pyfunc.load_model()`.
+This loaded PyFunc model can only be scored with a DataFrame input.
```

Comment:
Can we provide an overview of the accepted DataFrame input columns and length (i.e. single-row DataFrames)?

Reply:
Added a table, a listing of what the values mean, and warnings for ensuring that it's a single row DF and that n_periods is not optional.